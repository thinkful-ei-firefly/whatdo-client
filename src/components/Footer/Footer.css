.Footer{
  /* height: 30px; */
  /* background-color: grey; */
  background: linear-gradient(to bottom,white,grey);
  height: 10vh;
}