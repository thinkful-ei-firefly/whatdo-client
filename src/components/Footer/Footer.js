import React from 'react'

import './Footer.css'

export default class Footer extends React.Component{

  render(){
    return(
      <footer className='footer'>
      <p>What Do | 2019  @ abcj</p>
          
     
      </footer>
    )
  }
}