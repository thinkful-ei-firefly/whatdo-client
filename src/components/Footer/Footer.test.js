import React from 'react'
import ReactDOM from 'react-dom'
import Footer from './Footer'
import renderer from 'react-test-renderer'

describe('<Footer />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Footer />, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  it('renders as expected', () => {
    const tree = renderer
    .create(<Footer />)

    expect(tree).toMatchSnapshot()
  })
})
