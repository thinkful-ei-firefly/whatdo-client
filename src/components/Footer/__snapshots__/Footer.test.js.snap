// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders as expected 1`] = `
<footer
  className="footer"
>
  <p>
    What Do | 2019  @ abcj
  </p>
</footer>
`;
